src/plugins/vis_augmenter/public/vega/helpers.ts

```diff
@@ -25,6 +25,8 @@ import {
   VisLayerTypes,
 } from '../';
 
+// Given any visLayers, create a map to indicate which VisLayer types are present.
+// Serialize to an array since ES6 Maps cannot be stringified.
```

Comment:
nit - I don't think `[...visibleVisLayers]` itself is actually a serialization operation - we're just creating an array that can be serialized...

Reply:
Yeah, maybe "conversion" is more appropriate here..